src/legacy/core_plugins/timelion/public/timelion_vis_fn.ts

```diff
@@ -60,7 +60,7 @@ export const getTimelionVisualizationConfig = (
       help: '',
     },
     interval: {
-      types: ['string', 'null'],
```

Comment:
@flash1293 I wonder if it's safe to remove `null` type, since there can be saved vis with empty interval.

Reply:
It should be caught by the `default` value.

Reply:
Actually, I couldn't even create a vis with `null` interval value. So I think it's a kind of legacy thing.
The interval could be the only string.